index.js

```diff
@@ -59,28 +106,55 @@ getWinnersByYear();
 Hint: Investigate your data to find "team initials"!
 Hint: use `.reduce` */
 
-function getCountryWins(/* code here */) {
-
-    /* code here */
-
+function getCountryWins(data, initials) {
+    const countries = data.map((ele, i) => {
+        if (data[i]['Home Team Goals'] > data[i]['Away Team Goals']) {
+            return data[i]['Home Team Initials'];
+        }else if (data[i]['Home Team Goals'] < data[i]['Away Team Goals']){
```

Comment:
if else statements are a lot of fun, but sometimes really confusing too aha -- great job!